client/protocols/backup.py

```diff
@@ -26,13 +26,24 @@ def run(self, messages):
             return
 
         message_list = self.load_unsent_messages()
-        message_list.append(messages)
+
+        # Messages from the current backup to send first
+        if self.args.submit:
+            subm_messages = [messages]
+        else:
+            subm_messages = []
 
         access_token = auth.authenticate(False)
         log.info('Authenticated with access token %s', access_token)
+        log.info('Sending unsent messages %s', access_token)
+
+        # Send submission
+        response = self.send_all_messages(access_token, subm_messages, current=True)
```

Comment:
In the event of a backup this list is empty and just returns. Going to refactor this now.